doc/classes/Camera2D.xml

```diff
@@ -361,6 +363,7 @@
 			Right scroll limit in pixels. The camera stops moving when reaching this value.
 		</member>
 		<member name="limit_smoothed" type="bool" setter="set_limit_smoothing_enabled" getter="is_limit_smoothing_enabled">
+			If [code]true[/code] smooth camera when reaching camera limits.
```

Comment:
This feels incomplete. I think it needs an explanation of what "smooth camera" means.